finder-app/writer.c

```diff
@@ -0,0 +1,48 @@
+// Name 	: Sricharan
+// File 	: Writer.c
+// Brief	: Perform a write function that writes a string to the 			  specified file name.
+// course	: ECEN 5713 Advanced Embedded Software Development
+// Date	: 4th September 2022
+
+#include<sys/types.h>
+#include<sys/stat.h>
+#include<fcntl.h>
+#include<string.h>
+#include<syslog.h>
+#include<stdio.h>
+#include<unistd.h>
+// Argument - 1 = file name with path
+// Argument - 2 = string name to be added to the file name
+int main(int argc, char **argv){
+  // Syslog is logger statement used to log and not as time consuming as 	 printf
+  syslog(LOG_USER,"File Writing");
+  /* Arguments should be 3, 
+     1 - current file path (default)
+     2 - Expected file with the file path (1st explicit argument)
+     3 - Expected string to added inside the file
+  */   
+  if(argc != 3){
+     syslog(LOG_ERR,"Wrong number of arguments");
```

Comment:
@srki3050 think about how you can help the user do the right thing with the log message here.